src/color.rs

```diff
@@ -23,35 +23,35 @@ pub const ALPHA_OPAQUE: NormalizedF32 = NormalizedF32::ONE;
 
 /// A 32-bit RGBA color value.
 ///
-/// Byteorder: ABGR
+/// Byteorder: RGBA (relevant for bytemuck casts)
```

Comment:
I don't think that the _relevant for bytemuck casts_ is relevant. Most people have no idea what `bytemuck` is.

Reply:
The only way you can get at the internal representation is through casting to bytes or using the get function, which I removed in the PR. If you couldn't get at the internal representation at all, then the byteorder wouldn't need to be guaranteed/documented.